Sources/Plasma/PubUtilLib/plDrawable/plWaveSet7.cpp

```diff
@@ -4325,7 +4325,7 @@ void plWaveSet7::IInitGraph(int iPass)
 
 void plWaveSet7::IShuffleDownGraphs(int iPass)
 {
-    int i;
+    unsigned int i;
     for( i = iPass+1; i < kGraphShorePasses; i++ )
```

Comment:
`iPass` is a signed `int`, so theoretically there could still be overflow weirdness here. The proper fix would be to update the entire call chain to use `unsigned int` or `size_t` for this index.